dataflow/src/main/java/org/checkerframework/dataflow/cfg/ControlFlowGraph.java

```diff
@@ -55,10 +55,18 @@
     protected final UnderlyingAST underlyingAST;
 
     /**
-     * Maps from AST {@link Tree}s to sets of {@link Node}s. Every Tree that produces a value will
-     * have at least one corresponding Node. Trees that undergo conversions, such as boxing or
-     * unboxing, can map to two distinct Nodes. The Node for the pre-conversion value is stored in
-     * treeLookup, while the Node for the post-conversion value is stored in convertedTreeLookup.
+     * Maps from AST {@link Tree}s to sets of {@link Node}s.
+     *
+     * <ul>
+     *   <li>Most Trees that produce a value will have at least one corresponding Node.
+     *   <li>Trees that undergo conversions, such as boxing or unboxing, can map to two distinct
+     *       Nodes. The Node for the pre-conversion value is stored in {@link #treeLookup}, while
+     *       the Node for the post-conversion value is stored in {@link #convertedTreeLookup}.
+     * </ul>
+     *
+     * Some of the mapped-to nodes (in both maps) do not appear in {@link #getAllNodes} because
```

Comment:
I was confused about which maps the "both maps" was referring to.  I think it's `treeLookup` and `convertedTreeLookup`.